src/App.jsx

```diff
@@ -59,9 +60,9 @@ function App() {
         <Route element={<Sidebar />}>
           <Route path="/" element={<Feed />} />
           <Route path="/home" element={<HomePage />} />
-          <Route path="/profile" element={<ProfilePage />} />
-          <Route path="/profile/:handle" element={<ProfilePage />} />
-          <Route path="/edit" element={<EditProfile />} />
+          <Route path="/profile" element={<ProfilePage address={account}/>} />
+          <Route path="/profile/:handle" element={<ProfilePage address={account}/>} />
```

Comment:
No need to pass a prop on this one. It will be for profile search, so we won't display the profile of the current user. Actually having props.address == undefined can be a nice way to know if we are displaying the page of the current user or someone else's page (no button edit profile but a follow button)